src/Rules/ForbiddenMultipleClassLikeInOneFileRule.php

```diff
@@ -32,18 +34,11 @@ public function __construct(
         $this->nodeFinder = new NodeFinder();
     }
 
-    /**
-     * @return class-string<Node>
-     */
     public function getNodeType(): string
     {
         return FileNode::class;
     }
 
-    /**
-     * @param FileNode $node
-     * @return string[]
-     */
     public function processNode(Node $node, Scope $scope): array
```

Comment:
The `@param` should remain, so IDE can fill the mtehods on `$node`

Reply:
My lattest PHPStorm still doesn't. What IDE do you use?

![Screenshot from 2024-11-13 12-16-33](https://github.com/user-attachments/assets/05ed3c2d-18f9-4099-a27b-5af226430dad)

Reply:
This is why: https://github.com/symplify/phpstan-rules/pull/144#pullrequestreview-2435518909